src/plugins/controls/public/control_group/embeddable/control_group_container.tsx

```diff
@@ -252,10 +252,10 @@ export class ControlGroupContainer extends Container<
     this.subscriptions.add(
       this.getInput$()
         .pipe(
-          skip(1),
           distinctUntilChanged(
             (a, b) => Boolean(a.showApplySelections) === Boolean(b.showApplySelections)
-          )
+          ),
+          skip(1)
```

Comment:
@Heenawter change was Intended. I noticed a bug where this subscription would publish filters the first time the time range was changed on a Dashboard. This could nullify @nreese's change that stopped panels from querying when the Dashboard time range changed but they were using their own panel time range.

This change fixes that by properly skipping the initial instance on subscription from the `behaviorSubject`.